core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -28,7 +28,10 @@ use {
     solana_cost_model::cost_model::CostModel,
     solana_measure::measure_us,
     solana_runtime::{bank::Bank, bank_forks::BankForks},
-    solana_runtime_transaction::instructions_processor::process_compute_budget_instructions,
+    solana_runtime_transaction::{
```

Comment:
We could eliminate one compute budget calculation by using the compute budget details inside runtime tx inside `buffer_packets`